### ISSUE TITLE
[Feature Request] : add options for `Window.skip_taskbar` , `Window.always_on_top` and `Windows.utlity`

### ISSUE THREAD

**Issue by T-Dynamos** [kivy/kivy#8010]
Reference: https://wiki.libsdl.org/SDL_WindowFlags

Want these
![IMG_20220918_140704](https://user-images.githubusercontent.com/68729523/190893389-ba23cf9e-5b24-47d1-9a2d-5e9ff6f90bef.jpg)

Because I am making a taskbar [KivBar](https://github.com/T-Dynamos/KivBar)

**Comment by misl6** [comment]
For the records:
- https://github.com/kivy/kivy/pull/8142 adds support for `SDL_WINDOW_SKIP_TASKBAR`
- https://github.com/kivy/kivy/pull/8076 adds support for `SDL_WINDOW_ALWAYS_ON_TOP` via `SDL_SetWindowAlwaysOnTop`

**Comment by Julian-O** [comment]
So, two down, `SDL_Window_Utility` to go? Is that feature actually useful?

**Comment by T-Dynamos** [comment]
Yes it is, Suppose you have your taskbar made in kivy, you want to treat that window to be utility.

**Comment by p0lygun** [comment]
@Julian-O it is primarily used to set window decorations on X11 and is also used by various window managers to know how to handle the window, we should add https://github.com/kivy/kivy/labels/easy  and   https://github.com/kivy/kivy/labels/Component%3A%20core-app  tag to this issue

**Comment by ricleta** [comment]
Hey! I think I finished adding `SDL_Window_Utility`. I opened a PR already. #8940
